build-system/compile/log-messages.js

```diff
@@ -37,7 +37,8 @@ const formats = {
 };
 
 /** @return {!Promise<!Array<!Object>>} */
-const extractedItems = () => fs.readJson(extractedPath).then(Object.values);
+const extractedItems = async () =>
+  fs.readJson(extractedPath).then(Object.values);
```

Comment:
Can we rewrite `then(Object.values)` as well? Would something like this work?

```suggestion
const extractedItems = async () => Object.values(await fs.readJson(extractedPath));
```

Reply:
I saw this, but actually find the `then` form here much more readable than the nested `await`, so for now I'm leaving it as-is. If you feel strongly I can address in the upcoming follow-up to this PR